config/network.go

```diff
@@ -2,24 +2,48 @@ package config
 
 import (
 	"encoding/base64"
+	"errors"
 	"fmt"
 	"os"
 	"strings"
 
-	"errors"
-
 	"github.com/pelletier/go-toml/v2"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
 )
 
 const (
 	Base64NetworkConfigEnvVarName = "BASE64_NETWORK_CONFIG"
 )
 
+type ForkConfig struct {
+	URL              string `toml:"url"`          // URL is the URL of the node to fork from
+	BlockNumber      int64  `toml:"block_number"` // BlockNumber is the block number to fork from
+	BlockTime        int64  `toml:"block_time"`
+	Retries          int    `toml:"retries"`
+	Timeout          int64  `toml:"timeout"`
+	ComputePerSecond int64  `toml:"compute_per_second"`
+	RateLimitEnabled bool   `toml:"rate_limit_enabled"`
+}
+
+// NetworkConfig is the configuration for the networks to be used
 type NetworkConfig struct {
-	SelectedNetworks []string            `toml:"selected_networks"`
-	RpcHttpUrls      map[string][]string `toml:"RpcHttpUrls"`
-	RpcWsUrls        map[string][]string `toml:"RpcWsUrls"`
-	WalletKeys       map[string][]string `toml:"WalletKeys"`
+	SelectedNetworks []string `toml:"selected_networks"`
+	// EVMNetworks is the configuration for the EVM networks, key is the network name as declared in selected_networks slice.
+	// if not set, it will try to find the network from defined networks in MappedNetworks under known_networks.go
+	EVMNetworks map[string]*blockchain.EVMNetwork `toml:"evm_networks,omitempty"`
```

Comment:
should't this and the next one be uppercased? how should TOML look for them?  can you share an exnaple?

Reply:
Please refer to the test TestVariousNetworkConfig